src/app/pages/homepage/homepage.component.html

```diff
@@ -49,35 +49,31 @@
         <a class="button-ask">Ask Mentor</a>
       </div>
       <div class="news-block">
+        <h3>News</h3>
         <ul>
-          <li>
-            <i>19-03-1974</i>
-            <h3>Title</h3>
-            <p>Lorem ispum dolor sit amet
-              Lorem ispum dolor sit amet </p>
-            <a>More</a>
-          </li>
-          <li>
-            <i>19-03-1974</i>
-            <h3>Title</h3>
-            <p>Lorem ispum dolor sit amet
-              Lorem ispum dolor sit amet </p>
-            <a>More</a>
+          <li *ngFor="let news of messageService.newsBlock">
+            <i>{{ news.timestamp }}</i>
+            <p>{{ news.text }}</p>
           </li>
         </ul>
+        <a *ngIf="this.adminUserId" (click)="openInput()">New News</a>
+        <div *ngIf="newsInputOpen" class="news-input">
+          <input type="text" [(ngModel)]="messageService.chat.newsMessage">
+          <button (click)="messageService.sendNews(messageService.chat.newsMessage)" (click)="newsInputOpen = false">Add news</button>
```

Comment:
Don't invoke service in such situation. Reason: input can be empty and someone can create planty empty news. Better, create function in controller, check if new text is valid and do "sendNews" there.